src/sagemaker_mxnet_container/serving.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import logging
+import os
+
+import mxnet as mx
+from sagemaker_containers.beta.framework import encoders, env, modules, transformer, worker
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_MODEL_NAME = 'model'
+DEFAULT_MODEL_FILENAMES = {
+    'symbol': 'model-symbol.json',
+    'params': 'model-0000.params',
+    'shapes': 'model-shapes.json',
+}
+
+
+def default_model_fn(model_dir, preferred_batch_size=None):
```

Comment:
Does this value get populated with something other than None? I think it was 1 originally or takes the value from the environment class, but that was passed in directly originally.

https://github.com/aws/sagemaker-mxnet-container/blob/master/src/mxnet_container/serve/transformer.py#L110